### PR TITLE
fix: removed --dry-run

### PR DESCRIPTION
**Pull request by siwamsingh**
this pull request fixes issue #208 
removed "--dry-run" from cli.py and documentation

### CODE REVIEW

**Comment by siwamsingh** [comment]
@jdangerx  i made this pull request but its stuck  😅
 Its my second time contributing can you please check what happened .

File: src/pudl_archiver/cli.py
Comment (jdangerx) [inline src/pudl_archiver/cli.py:1]: While this is where the `--dry-run` argument is passed in, we still need to update all the downstream references - see how `main()` calls 
```python
    asyncio.run(archive_datasets(**vars(args)))
```

And then `archive_datasets` takes a `dry_run` parameter?

**Review by jdangerx — CHANGES_REQUESTED** [review]
Thanks for the PR! There's still some work left to do since you have to chase down all the downstream references to `dry_run`.

**Comment by jdangerx** [comment]
Hi @siwamsingh !  You closed this - did you still want to try to merge this in?

**Comment by siwamsingh** [comment]
Sorry @jdangerx  i am not able to work due to some personal problem  😥 . plz assign this issue to someone else i am really sorry.

**Pull request closed without merging**